scss/utils/helpers/_customPropertyValue.scss

```diff
@@ -0,0 +1,18 @@
+////
+/// @group Responsive Variables
```

Comment:
Should we call this `@group Custom Properties`? The mixin may in the future be used with custom properties other than responsive vars